README.md

```diff
@@ -0,0 +1,108 @@
+# Spring reCaptcha
+
+Library providing a custom filter to validate reCaptcha
+
+## Requirements
+
+* JDK 8 +
+* Spring Boot MVC application
```

Comment:
Actually I didn't used the SB 2.2 feature about configuration. I think it's valuable for multiple property files but here I only have one. But still it uses spring boot 2.2 dependencies so as we can insure that it will work with spring boot 2.1 I modified it

Reply:
Since the application is using a few features only present in SB 2.2 (`proxyBeanMethods` and `@ConstructorBinding`), it's certain to not work with SB < 2.2.

Thanks for the modification :)

Reply:
Indeed :sweat_smile: